src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/System.Net.Http.WinHttpHandler.Functional.Tests.csproj

```diff
@@ -21,6 +21,9 @@
     <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers.cs">
       <Link>Common\System\Net\HttpTestServers.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers2.cs">
+      <Link>Common\System\Net\HttpTestServers2.cs</Link>
```

Comment:
It's currently in System.Net.Tests namespace in terms of the actual static class in the file. The `HttpTestServer2.cs` file is the same as the `HttpTestServer.cs` file.

I will address this potentially when we merge these 2 files.